KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -673,7 +695,7 @@
 
 "kh_uisdk_meeting_point" = "Meeting point";
 
-"kh_uisdk_metered" = "Metered";
+"kh_uisdk_metered" = "Metered price";
```

Comment:
@AleksanderWedrychowskiKarhoo did you change this in phrase? Or was it Olivier?

Reply:
I changed it with @olivierkarhoo's approval.